blog/vr-volume-viewer.md

```diff
@@ -60,6 +60,5 @@ This is a demo of the temperature and salinity of the Rhode Island narragansett
 <p align="center"><img src="/content/images/blog/vr-volume-viewer/image5.gif"/></p>
 
 ### Conclusion
-It is important to do research on the type of tools that could be used to easily visualize 3D data. However, it is vital to understand the rendering techniques and algorithms that those tools work with. Being able to implement ray marching helped our team to quickly develop a tool that adjusted to our own requirements and we can maintain following software engineer standards.
-
+It is important to research the types of tools available for easily visualizing 3D data. However, it's equally vital to understand the rendering techniques and algorithms these tools employ. Gaining an understanding of, and being able to implement, ray marching enabled our team to quickly develop a tool that meets our specific needs. This tool can be maintained according to established software engineering standards.
```

Comment:
remove the comma after "implement". otherwise, looks great!

Reply:
@ashleysyg  I applied your suggestions. Please review it.